ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -564,26 +565,52 @@ private Optional<List<Request>> extractRequests(final Optional<List<String>> may
                 .collect(Collectors.toList()));
   }
 
-  private void logImportedBlockInfo(final Block block, final int blobCount, final double timeInS) {
+  private void logImportedBlockInfo(
+      final Block block,
+      final int blobCount,
+      final double timeInS,
+      final Optional<Integer> nbParallelizedTransations) {
     final StringBuilder message = new StringBuilder();
-    message.append("Imported #%,d / %d tx");
+    final int nbTransactions = block.getBody().getTransactions().size();
+    message.append("Imported #%,d  (%s)|%5d tx");
     final List<Object> messageArgs =
         new ArrayList<>(
-            List.of(block.getHeader().getNumber(), block.getBody().getTransactions().size()));
+            List.of(
+                block.getHeader().getNumber(),
+                shortenString(block.getHash().toHexString()),
+                nbTransactions));
     if (block.getBody().getWithdrawals().isPresent()) {
-      message.append(" / %d ws");
+      message.append("|%3d ws");
```

Comment:
wonder if we should remove withdrawals since they are always 16?

Reply:
I think we should keep it, in case there is an issue, also, it is part of the block, I don't think we should things implicit at this level.

Reply:
There's plenty of the block data we already omit. IMO the logs should be optimised for readability, one aspect of which is brevity and this adds to the noise IMO. I take the point about if there were problems, but there hasn't and seems unlikely there error would be - I would argue exceptions and DEBUG should be used for problems, whereas this log is supposed to inform about the happy path blocks (and 16 ws gives no new information ~100% of the time). The one time it was useful was during the Shanghai fork :)